src/interfaces/person-interface.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * A person object from the API. See:
+ * https://github.com/bcgov/business-schemas/blob/master/src/registry_schemas/schemas/person.json
+ */
+
+export interface PersonIF {
+  givenName?: string
+  familyName?: string
+  additionalName?: string
+  middleInitial?: string
+  email?: string
+  [propName: string]: any // excess properties
+}
+
+export const EmptyPersonIF: PersonIF = {
+  givenName: '',
+  familyName: '',
+  additionalName: null
```

Comment:
This is probably OK but watch out for the following:
- you're defining some optional properties
- you're inconsistently defining some string properties as empty strings, and other as null
- the remaining properties are undefined, meaning that changes to them will not trigger reactivity 

What to define in this Empty Person object depends on how it's used, which is subject not objective, which is not ideal. I'd like to define a best practise for such empty objects. Any thoughts?

Obviously, required properties should be defined. There aren't any here.

Should all optional properties be defined?

Should primitive types (eg, string, number) be assigned empty values or null?

Reply:
I think I'll leave this out of `bcrs-shared-components`

Reply:
Object has been removed. This conversation is now obsolete.